# A grasping point defining the grasping position and the surface's normal at the grasping point
std_msgs/Header header
int32 label
geometry_msgs/Point position
geometry_msgs/Point normal